visualization.goom/resources/settings.xml

```diff
@@ -18,6 +18,10 @@
           </constraints>
           <control type="spinner" format="string" />
         </setting>
+        <setting id="use_pixel_buffer_objects" type="boolean" label="30007" help="30008">
```

Comment:
You need then also to add on `resources/language/resource.language.en_gb/strings.po`, e.g. :smile::
```po
msgctxt "#30007"
msgid "Pixel buffer objects"
msgstr ""

msgctxt "#30008"
msgid "To increase render performance"
msgstr ""
```
my english not the best, but something like this then needed.

Reply:
Sorry, my mistake, forgot to test previous changes. Added strings to 'strings.po'. All should be good now.